static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js

```diff
@@ -61,7 +61,13 @@ export const acquisitionsBannerFivTemplate = (
                     ${mobileButton}${arrowWhiteRight.markup}
                 </a>
                 <div class="engagement-banner__payment-logos engagement-banner__payment-logos--fiv">
-                    ${paymentMethods.markup}
+                    <img
+                        src="${config.get(
+                            'images.acquisitions.paypal-and-credit-card',
+                            ''
+                        )}"
+                        alt="payment options"
```

Comment:
Small accessibility request: can we change this to "Accepted payment methods: Visa, Mastercard, American Express and Paypal". Or something similar?

Reply:
👍